packages/core/src/common/keybinding.ts

```diff
@@ -192,29 +205,43 @@ export class KeybindingRegistry {
     }
 
     /**
-     * Get the list of keybindings matching keyCode.  The list is sorted by
-     * priority (see #sortKeybindingsByPriority).
+     * Get the lists of keybindings matching fully or partially matching a KeySequence.
+     * The lists are sorted by priority (see #sortKeybindingsByPriority).
      *
-     * @param keyCode The key code for which we are looking for keybindings.
+     * @param keySequence The key sequence for which we are looking for keybindings.
      */
-    getKeybindingsForKeyCode(keyCode: KeyCode): Keybinding[] {
-        const result: Keybinding[] = [];
+    getKeybindingsForKeySequence(keySequence: KeySequence): { full: Keybinding[], partial: Keybinding[] } {
```

Comment:
for clients, it would be good to have a named type

Reply:
Yes I hesitated about that.. I'll add this then.